.github/workflows/codeql.yml

```diff
@@ -16,8 +16,6 @@ permissions:
 
 jobs:
   codeql-analyze:
-    # Run only on pull requests, see https://github.com/github/codeql-action/issues/1537
-    if: ${{ github.event_name == 'pull_request' }}
```

Comment:
this is here for a reason (described in the linked issue), you are not fixing the reason this override was added